toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Authenticator.kt

```diff
@@ -51,8 +51,65 @@ import com.arcgismaps.httpcore.authentication.OAuthUserSignIn
 @Composable
 public fun Authenticator(
     authenticatorState: AuthenticatorState,
-    modifier: Modifier = Modifier.fillMaxSize(),
+    modifier: Modifier = Modifier,
     onPendingOAuthUserSignIn: ((OAuthUserSignIn) -> Unit)? = null
+) {
+    AuthenticatorImpl(
+        authenticatorState = authenticatorState,
+        modifier = modifier,
```

Comment:
We should do this to get fullscreen behavior
```suggestion
        modifier = modifier.fillMaxSize(),
```